### ISSUE TITLE
Accessibility feedback: cluttered interface

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#3691]
Reporting some first feedback from assistive technologies users, see https://make.wordpress.org/accessibility/2017/11/28/screen-reader-user-experience-with-gutenberg/

first user:
> There is way to much going on and far too many menus to deal with. With the current version I can hit the f key 3 times and get to the Title, now it takes 3 times as much if I manage to land on it after trying to add something. I find it very difficult to accomplish anything because of the many menus and options. Please let the option for the “Classic editor” stay.

second user:
> There are a lot of buttons a lot unlabeled. I found Gutenberg full of clutter, buttons and the like.

This was in some way already mentioned on other issues. Ideally, the interface should be simplified as much as possible and should show controls only when needed. Reducing as much as possible the tab stops is a must for keyboard navigation.

#3195 could help with this, and ideally the "navigation mode" should be the default for keyboard users.

**Comment by afercia** [comment]
Moving this feedback to #3195

**Comment by afercia** [comment]
Reopening for history and because #3195 was closed.

**Comment by mtias** [comment]
Closing as navigation mode has been introduced in #16500. We should continue to iterate on this.
